src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -1200,6 +1200,17 @@ private static SyntaxNode WithAttributeLists(SyntaxNode declaration, SyntaxList<
             _ => declaration,
         };
 
+    internal override ImmutableArray<SyntaxNode> GetPrimaryCtor(SyntaxNode declaration)
```

Comment:
not sure why this is returning an array.

Reply:
Change to "SyntaxNode?"